packages/firestore/src/local/simple_db.ts

```diff
@@ -162,10 +162,20 @@ export class SimpleDb {
     // ua = 'Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML,
     // like Gecko) Chrome/39.0.2171.71 Safari/537.36 Edge/12.0';
 
+    // iOS Safari: Disable for users running iOS version < 10.
+    const iOSVersion = SimpleDb.getIOSVersion(ua);
+    const isUnsupportedIOS = 0 < iOSVersion && iOSVersion < 10;
+
+    // Checks whether IndexedDB is supported on the browser
+    const noIndexedDB =
```

Comment:
It is redundant since we already check it at the top of the function.

Reply:
Thanks for catching that